app/controllers/application_controller.rb

```diff
@@ -1,3 +1,27 @@
 class ApplicationController < ActionController::Base
+  helper_method :current_user
+  before_action :login_required
+  protect_from_forgery with: :exception
+
+  add_flash_types :success, :info, :warning, :danger
+
   http_basic_authenticate_with :name =>ENV['BASIC_AUTH_USERNAME'], :password => ENV['BASIC_AUTH_PASSWORD'] if Rails.env == "production"
+
+  private
+
+  def change_layout
+    if current_user.admin?
+      layout "admin/admin_application"
+    else
+      layout "admin_application"
+    end
```

Comment:
この部分は三項演算子で記載することも可能ですが、分かりづらければこのままで問題ないです。
パターンが3つ以上の場合は`if, elseif`文で大丈夫です。
```suggestion
    current_user.admin? ? layout "admin/admin_application" : layout "admin_application"
```

Reply:
0981c04 55296fa にて修正しました。三項演算部分はcontrollerにて書いてありますので、消し忘れでした。すみません。